src/ray/raylet_client/raylet_client.h

```diff
@@ -29,6 +29,12 @@
 #include "src/ray/protobuf/common.pb.h"
 #include "src/ray/protobuf/gcs.pb.h"
 
+#ifdef _WIN32
```

Comment:
Actually maybe put it in `ray/util/process.h` that should be included by everybody who needs it :)